Sprint-2/debug/2.js

```diff
@@ -12,3 +12,5 @@ console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
 // This program should tell the user the last digit of each number.
 // Explain why getLastDigit is not working properly - correct the problem
+
+//I had to make a parameter for the function to take in instead of having a constant value that it uses, I made the const let and I added the parameter num to the getLast Digit Function
```

Comment:
I did not have a full understanding of the requirements I guess. I see now that the purpose was not to reassign the variable but to understand the scope of the variable and how it can be manipulated within or outside of a function.

I've adjusted my code to fit those requirements

Reply:
I don't know the actual requirements of this exercise either. I just want you to understand why the `num` inside the function is not related to the `num` declared outside the function so that you can understand changing `const num = 103;` to `let num = 103` has no effect in this exercise.